docs/core/compatibility/sdk/10.0/prune-packagereference-privateassets.md

```diff
@@ -58,6 +58,9 @@ The same project configuration now generates a *.nuspec* file with the prunable
 </dependencies>
 ```
 
+> [!NOTE]
+> Another change was made in .NET 10 RC1 that narrows the scope of this breaking change somewhat. Now, prunable dependencies are removed only for projects with a `net10.0` target framework moniker (TFM).
```

Comment:
I think these are 2 separate concepts.

- Prunable dependencies are privatized whenever pruning is enabled.
- Pruning is enabled by default for projects targeting .NET 10, including multi targeting. 

Maybe we should change the example above to call out .NET 10 in order to not create confusion?

Reply:
NU1510 is tfm bound because it's a warning, build breaking. 
Privatization is how pruning works, so it'll raised even on .NET 9.